### PR TITLE
Removed Dashboard link from navbar

### PR DESCRIPTION
**Pull request by HarshwardhanSingh**
This PR is about adding some links in the header which were discussed in sprint planning #1. 
No other branch depends on this so you can merge this as well.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -11,11 +11,10 @@
     <nav class="navbar navbar-default navbar-fixed-top">
       <div class="container">
         <div class="navbar-header">
-          <%= link_to "Leave Tracker", root_path, class: 'navbar-brand' %>
+          <%= link_to "Leave Tracker", dashboard_path, class: 'navbar-brand' %>
```
Comment (iffyuva) [inline app/views/layouts/application.html.erb]: its better to make dashboard served at `root_path`

Comment (HarshwardhanSingh) [inline app/views/layouts/application.html.erb]: @iffyuva I was thinking of serving a welcome page at root path.

Comment (iffyuva) [inline app/views/layouts/application.html.erb]: if user hasn't logged in, show welcome page, else show dashboard.

Comment (HarshwardhanSingh) [inline app/views/layouts/application.html.erb]: @iffyuva okay, will fix that.

**Comment by deepak** [comment]
@HarshwardhanSingh do you want to use `root_path` in sessions_controller where you have used `dashboard_path` ?

**Comment by HarshwardhanSingh** [comment]
Since both of then redirect to pages#dashboard yes, I can.
Should I?

**Comment by deepak** [comment]
@HarshwardhanSingh as discussed we can, because:

1. intent is better expressed. for login we do not care specifically which page to redirect to. we just want to redirect away from that page to a landing page. the root page is good for that
2. dashboard can change in future. but there will always be a root_path. so root path is good if we do not care specifically that it has to be the dashboard page

**Comment by deepak** [comment]
@HarshwardhanSingh looks good. please rebase and merge PR once done

**Pull request merged**